### PR TITLE
Switch accounts/orgs dropdown

### LINKED ISSUE #4200: New Feature: Multi-Org - Switch Accounts & Switch Orgs
**ETA:** Aug 18 2022
**PR:** https://github.com/influxdata/ui/issues/3968

The user will have the ability to:
- Switch accounts from a new global header at the top of the screen
- Switch organizations from a new global header at the top of the screen
- Navigate to their user profile from a new global header at the top of the screen

Users will no longer have to navigate to account > settings in order to switch accounts

### PR DESCRIPTION
**Pull request by noramullen1**
Closes #4200

### CODE REVIEW

File: content/influxdb/cloud/account-management/switch-org.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    name: Switch InfluxDB accounts
+    parent: Account management
+weight: 105
+---
+If you belong to more than one {{< cloud-name >}} organization with the same email address, you can switch from one organization to another while staying logged in. An account can contain multiple organizations. 
+
+To switch {{< cloud-name "short" >}} organizations:
+
+1. In the {{< cloud-name "short" >}} UI, click the organization name dropdown in the upper left and select **Settings**.
+2. Click **Switch Organization**. If this option doesn't appear, your email address is only associated with one organization.
```
Comment (noramullen1) [inline content/influxdb/cloud/account-management/switch-org.md:17]: @gunnaraasen -- Can you confirm these steps? wasn't able to try it out with multiple orgs.

File: content/influxdb/cloud/account-management/data-usage.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ View the statistics of your data usage and rate limits (reads, writes, and delet
 
 To view your {{< cloud-name >}} data usage, do the following:
 
-1. Click the **user avatar** on the left-side navigation.
+1. Click the organization name dropdown in the upper left.
```
Comment (kelseiv) [inline content/influxdb/cloud/account-management/data-usage.md:23]:
```suggestion
1. Click the organization name drop-down list in the upper left.
```

Comment (kelseiv) [reply content/influxdb/cloud/account-management/data-usage.md:23]: or [list](https://developers.google.com/style/word-list) alone

File: content/influxdb/cloud/account-management/offboarding.md [hunk 3]
```diff
@@ -65,11 +65,9 @@ However, those in multi-user organizations must contact
 [InfluxData Support](mailto:support@influxdata.com) to delete your organization. 
 {{% /note %}}
 
-1. Click the **user avatar** in the top right corner of your {{< cloud-name "short" >}}
+1. 1. Click the account name dropdown in the upper left of your {{< cloud-name "short" >}}
```
Comment (kelseiv) [inline content/influxdb/cloud/account-management/offboarding.md:68]:
```suggestion
1. Click the account name drop-down list in the upper left of your {{< cloud-name "short" >}}
```

File: content/influxdb/cloud/account-management/pricing-calculator.md [hunk 4]
```diff
@@ -34,7 +34,7 @@ Guidelines used to estimate costs for default configurations:
 
 1. Do one of the following:
 
-  - Free Plan. Click the **user avatar** in the top right corner of your
+  - Free Plan. Click the organization name dropdown in the upper left of your
```
Comment (kelseiv) [inline content/influxdb/cloud/account-management/pricing-calculator.md:37]:
```suggestion
  - Free Plan. Click the organization name drop-down list in the upper left of your
```

File: content/influxdb/cloud/account-management/switch-account.md [hunk 5]
```diff
@@ -15,9 +15,12 @@ You can also set a default account. The default account is the account automatic
 
 To switch {{< cloud-name "short" >}} accounts:
 
-1. In the {{< cloud-name "short" >}} UI, select the **user avatar** in the left
-   navigation menu, and select **Account** > **Settings**.
-2. Click **Switch Account**. If this option doesn't appear, your email address is only associated with one account.
-3. Select the account you want to switch to or set as the default in the window that appears.
-4. To switch to the account, select **Switch Account**.
-5. To set an account as the default, select **Set Default Account**.
+1. In the {{< cloud-name "short" >}} UI, click the account name dropdown in the upper left.
```
Comment (kelseiv) [inline content/influxdb/cloud/account-management/switch-account.md:18]:
```suggestion
1. In the {{< cloud-name "short" >}} UI, click the account name drop-down list in the upper left.
```

File: content/influxdb/cloud/account-management/switch-account.md [hunk 6]
```diff
@@ -15,9 +15,12 @@ You can also set a default account. The default account is the account automatic
...
+2. Select **Switch Account**. If this option doesn't appear, your email address is only associated with one account.
+3. Select the account you want to switch to.
+
+To set an account as the default:
+
+1. Click the user avatar in the upper-right and select **Profile**.
+2. Select a **Default Account** from the dropdown.
```
Comment (kelseiv) [inline content/influxdb/cloud/account-management/switch-account.md:25]:
```suggestion
2. Select a **Default Account** from the drop-down list. 
```

File: content/influxdb/cloud/account-management/switch-org.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+  influxdb_cloud:
+    name: Switch InfluxDB accounts
+    parent: Account management
+weight: 105
+---
+If you belong to more than one {{< cloud-name >}} organization with the same email address, you can switch from one organization to another while staying logged in. An account can contain multiple organizations. 
+
+To switch {{< cloud-name "short" >}} organizations:
+
+1. In the {{< cloud-name "short" >}} UI, click the organization name dropdown in the upper left and select **Settings**.
```
Comment (kelseiv) [inline content/influxdb/cloud/account-management/switch-org.md:16]:
```suggestion
1. In the {{< cloud-name "short" >}} UI, click the organization name drop-down list in the upper left and select **Settings**.
```

**Review by kelseiv — COMMENTED** [review]
@noramullen1 one minor global edit -- fyi, I turned on the feature flag, and saw some odd characters, so didn't test steps

File: content/influxdb/cloud/account-management/switch-org.md (hunk 1 above)
Comment (ALuckette) [reply content/influxdb/cloud/account-management/switch-org.md:17]: This is incorrect. Steps are: 

To switch {{< cloud-name "short" >}} organizations:

1. In the {{< cloud-name "short" >}} UI, click the organization name dropdown in the upper left and select **Switch Organizations**.
2. Then select the organization that you would like to switch to. 

Note: If the organization is in a different account, you must switch accounts first in order to view the account's corresponding organizations. 

@noramullen1

File: content/influxdb/cloud/account-management/switch-org.md [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+seotitle: Switch between InfluxDB Cloud organizations
+description: >
+  Switch from one InfluxDB Cloud account to another and set a default account.  
+menu:
+  influxdb_cloud:
+    name: Switch InfluxDB accounts
+    parent: Account management
+weight: 105
+---
+If you belong to more than one {{< cloud-name >}} organization with the same email address, you can switch from one organization to another while staying logged in. An account can contain multiple organizations.
```
Comment (gunnaraasen) [inline content/influxdb/cloud/account-management/switch-org.md:12]: Let's add an intro on orgs:

> Organizations are isolated environments for time series data and the resources to make use of it, e.g. tasks, dashboards, etc. InfluxDB Cloud accounts can have one or more organizations. Organizations can only belong to a single InfluxDB Cloud account.

> If your InfluxDB Cloud user belongs to more than one {{< cloud-name >}} organization, you can switch from one organization to another using the following steps:

**Review by gunnaraasen — COMMENTED** [review]
This is looking good. I think the biggest thing is that we should refer to the header as the "header" instead of "upper left".

File: content/influxdb/cloud/account-management/switch-org.md (hunk 8 above)
Comment (noramullen1) [reply content/influxdb/cloud/account-management/switch-org.md:12]: Used the wording from our existing org docs instead.

**Pull request merged**